LocalAI/run.sh

```diff
@@ -4,7 +4,7 @@ set -ex
 
 K3S_MANIFEST_DIR=${K3S_MANIFEST_DIR:-/var/lib/rancher/k3s/server/manifests/}
 
-VERSION="1.0.2"
+VERSION="2.23.0"
```

Comment:
shouldn't this be the chart version? ([3.4.0](https://github.com/go-skynet/helm-charts/blob/274accb100c057b057a9b30c6ced1a8235dd291d/charts/local-ai/Chart.yaml#L6) in this case)

Reply:
@jimmykarily ohhh, new topic for me hahaha, so the one that needs to be bumped to 2.23.0 is this one? 

https://github.com/go-skynet/helm-charts/blob/274accb100c057b057a9b30c6ced1a8235dd291d/charts/local-ai/Chart.yaml#L2